packages/plugin-ext/src/main/browser/dialogs/modal-notification.ts

```diff
@@ -17,6 +17,7 @@ import {injectable} from 'inversify';
 import {Message} from '@phosphor/messaging';
 import {Key} from '@theia/core/lib/browser';
 import {AbstractDialog} from '@theia/core/lib/browser/dialogs';
+import {NOTIFICATION, ICON, TEXT} from '@theia/messages/lib/browser/notifications';
```

Comment:
I'm sorry, that should not be exported, nor reused. Please inline those styles for plugin-ext if you need something like this. They are subject to change in #1360.

BTW, this slipped in a review, as such constants should be encapsulated in a namespace. Also styles shouldn't be reused in such a way in general. You can e.g. have a different implementation for `MessageService` and replace the `messages` extension.

Reply:
Totaly agree.  I was confused with export there(thought about local rules)
https://github.com/theia-ide/theia/blob/master/packages/messages/src/browser/notifications.ts#L18